grails-app/assets/javascripts/customer/CustomerShowController.js

```diff
@@ -0,0 +1,91 @@
+function CustomerShowController() {
+        var _this = this;
+        var postalCodeValidator
```

Comment:
define o teu this.reference com teu elemento mais externo

Ai depois nao precisa fica buscando no escopo inteiro toda hora: document.getElementById('address')

Exemplo:
```
function BreachSecrecyAttachmentController() {
    this.reference = $(".js-breach-secrecy-attachment");

    var _this = this;

    this.init = function(uploadDoneBtnReference) {
        var uploadUrl = _this.reference.find(".js-temporary-upload-url").val();
```